manifest_staging/charts/aad-pod-identity/templates/nmi-psp-rolebinding.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "aad-pod-identity-psp.nmi.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "aad-pod-identity.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "aad-pod-identity-psp.nmi.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "aad-pod-identity-psp.nmi.fullname" . }}
```

Comment:
This should be NMI service account `{{ template "aad-pod-identity.nmi.fullname" . }}`

Reply:
done